Simplenote/Classes/TagView.swift

```diff
@@ -128,7 +128,10 @@ class TagView: UIView {
             return
         }
 
-        scrollView.scrollRectToVisible(addTagField.frame.insetBy(dx: -Constants.wrappingStackViewMargins.right, dy: 0),
+        layoutIfNeeded()
```

Comment:
This invocation here _feels_ off. But TBH I'm not sure where it could be otherwise!!

Reply:
Totally agree on that 🤔